src/stats.rs

```diff
@@ -21,9 +21,9 @@ pub struct EncoderStats {
   /// Stores count of pixels belonging to each transform type in this frame
   pub tx_type_counts: [usize; TX_TYPES],
   /// Stores count of pixels belonging to each luma prediction mode in this frame
-  pub luma_pred_mode_counts: [usize; PREDICTION_MODES],
+  pub luma_pred_mode_counts: Vec<usize>,
   /// Stores count of pixels belonging to each chroma prediction mode in this frame
-  pub chroma_pred_mode_counts: [usize; PREDICTION_MODES],
+  pub chroma_pred_mode_counts: Vec<usize>,
```

Comment:
why this?

Reply:
can't derive #[Debug] for arrays longer than 32

Reply:
I don't know why this derives debug anyway, maybe it shouldn't

Reply:
Probably Debug can be dropped.

Reply:
Unfortunately can't do that as Packet also derives Debug.